### PR TITLE
Specify PostgreSQL version correctly for Travis

### LINKED ISSUE #74: Tests are failing on Travis
...and it looks like they have been for a while.

See https://travis-ci.org/PyconUK/ironcage/builds/250375431 for instance.

Looks like this is the cause:

```
1.32s$ sudo service postgresql stop
 * Stopping PostgreSQL 9.1 database server        * Stopping PostgreSQL 9.2 database server        * Stopping PostgreSQL 9.3 database server        * Stopping PostgreSQL 9.4 database server       
cp: cannot stat `/var/lib/postgresql/9.6': No such file or directory
0.05s$ sudo service postgresql start 9.6
PostgreSQL 9.6 is not installed!
```

### PR DESCRIPTION
**Pull request by inglesp**
Fixes #74

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -12,4 +12,4 @@ install:
   - pip install -r requirements.txt
 
 addons:
-  postgresql: 9.6
+  postgresql: "9.6"
```
Comment (meshy) [inline .travis.yml]: [From the looks of it](https://docs.travis-ci.com/user/database-setup/#PostgreSQL) this also requires:

``` yaml
services:
  - postgresql
```

Comment (inglesp) [inline .travis.yml]: :heart:

**Comment by meshy** [comment]
Oh poop, it still failed...

**Comment by meshy** [comment]
Hm...

```
sudo service postgresql stop

 * Stopping PostgreSQL 9.2 database server        * Stopping PostgreSQL 9.3 database server        * Stopping PostgreSQL 9.4 database server       

cp: cannot stat `/var/lib/postgresql/9.6': No such file or directory

0.08s$ sudo service postgresql start 9.6

PostgreSQL 9.6 is not installed!
```

**Comment by meshy** [comment]
Aha! Postgres 9.6 is only available on the "Ubuntu Trusty" Travis build.

I think that can be changed in the preferences.... I'll have a look, and see if I can switch it over.

**Comment by meshy** [comment]
Ok, it looks like [we need to add](https://docs.travis-ci.com/user/trusty-ci-environment/)
```
dist: trusty
```

@inglesp happy to give that a shot?

**Comment by meshy** [comment]
Oh no! I meant to open a new branch and PR for that commit!

I hope you don't mind me committing to your branch here, @inglesp!

**Comment by meshy** [comment]
FWIW, it looks like that works, but the build is still failing due to some flake8 checks.

**Pull request merged**